Common/TableProducer/Converters/converter.cxx

```diff
@@ -0,0 +1,615 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   converter.cxx
+/// \since  2024-11-12
+/// \author Nicolò Jacazio nicolo.jacazio@cern.ch
+/// \brief  Combined converter task
+///
+
+#include <vector>
+#include <string>
+
+// O2 includes
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Common/DataModel/Multiplicity.h"
+
+// O2Physics includes
+#include "TableHelper.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::zdc;
+
+// Converts bc_000 into bc_001
+struct bcConverter {
+  Produces<aod::BCs_001> bc_001;
+  void init(o2::framework::InitContext& initContext)
+  {
+    doprocessConverter.value = isTableRequiredInWorkflow(initContext, "BCs_001");
+    if (doprocessConverter) {
+      LOG(info) << "Enabling converter for table BCs_001";
+    }
+  }
```

Comment:
This part is very similar in all converters. Can it be put written only once with the argument of the table name?